docs/metrics.md

```diff
@@ -7,8 +7,6 @@
 
 The Splunk Distribution of OpenTelemetry Java agent gathers basic application metrics.
 Because these splunk-specific metrics are still experimental they are not enabled by default.
```

Comment:
After this are there any more remaining Splunk-specific metrics left?

Reply:
There are profiling metrics.